app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_like_counter
+    post.increment!(:likescounter)
+  end
+end
```

Comment:
- [ ] [OPTIONAL] Like my previous suggestion, it would be nice to use the `after_save` Rails Active Callback here 👍